fashify/src/Hooks/FilterHook.js

```diff
@@ -0,0 +1,21 @@
+import { useData } from "../contexts/data-context";
+import {
+  categoryFilter,
+  priceRangeFilter,
+  ratingFilter,
+  sizeFilter,
+  sortByPrice,
+} from "../utils/utils";
```

Comment:
Here instead of using filename as utils, you can use filterFunctions or something like that, utils is generally name of the folder.

Reply:
Thnks for the suggestion